README.md

```diff
@@ -33,39 +33,41 @@ The system comprises the following components:
 ## Code Prerequisites
 
 In order to build and run the code, you need the following installed:
-  - [Node](https://nodejs.org/)
-  - [Yarn](https://yarnpkg.com/en/docs/install)
+  - [Node](https://nodejs.org/); latest LTS
```

Comment:
Let's specify the version here. We won't automatically support the latest LTS. It will confuse the reader too.